doc/modules/impute.rst

```diff
@@ -56,6 +57,23 @@ Note that, here, missing values are encoded by 0 and are thus implicitly stored
 in the matrix. This format is thus suitable when there are many more missing
 values than observed values.
 
+The :class:`SimpleImputer` class also supports categorical datas represented as
+string values or pandas categoricals when using the ``most_frequent`` or
+``constant`` strategy::
+
+    >>> import pandas as pd
+    >>> df = pd.DataFrame([["a", "x"],
+    ...                    ["", "y"],
+    ...                    ["a", ""],
+    ...                    ["b", "y"]], dtype="category")
+    ...
+    >>> imp = SimpleImputer(missing_values="", strategy="most_frequent")
```

Comment:
This example of course now illustrates the use of `missing_values` keyword as well, but personally I would use the more typical pandas use case where your missing values are already as np.nan 

I think the `missing_values` keyword would be more typical with integers